hubble/src/indexer/mod.rs

```diff
@@ -98,6 +220,69 @@ impl Default for FinalizerConfig {
             min_duration_between_monitor_checks:
                 FinalizerConfig::default_min_duration_between_monitor_checks(),
             retry_later_sleep: FinalizerConfig::default_retry_later_sleep(),
+            retry_error_sleep: FinalizerConfig::default_retry_error_sleep(),
+        }
+    }
+}
+
+#[derive(Clone, Debug, serde::Deserialize)]
+pub struct FixerConfig {
+    // sleep time (in seconds) when there is nothing to fix.
+    // default: 5 seconds
+    #[serde(
+        rename = "retry_later_sleep_seconds",
+        default = "FixerConfig::default_retry_later_sleep",
+        deserialize_with = "FixerConfig::deserialize_seconds"
+    )]
+    pub retry_later_sleep: Duration,
+
+    // sleep time (in seconds) when there is an error.
+    // default: 5 seconds
+    #[serde(
+        rename = "retry_error_sleep_seconds",
+        default = "FixerConfig::default_retry_error_sleep",
+        deserialize_with = "FixerConfig::deserialize_seconds"
+    )]
+    pub retry_error_sleep: Duration,
+
+    // maximum number of blocks to send in one message. An empty block is sent if no
+    // events are found after this amount of blocks.
+    // default: 1000
+    #[serde(
+        rename = "max_blocks_in_message",
+        default = "FixerConfig::default_max_blocks_in_message"
+    )]
+    pub max_blocks_in_message: u64,
+}
+
+impl FixerConfig {
+    pub fn default_retry_later_sleep() -> Duration {
+        Duration::from_secs(5)
+    }
+
+    pub fn default_retry_error_sleep() -> Duration {
+        Duration::from_secs(5)
+    }
+
+    pub fn default_max_blocks_in_message() -> u64 {
+        1000
+    }
```

Comment:
I would just inline these in the `impl Default for FixerConfig` as it’s realistically the only place where they’ll be used. less bloat